### PR TITLE
Improvements to VgDisplay for `vgdisplay -vv` command handling for 1.x branch

### PR DESCRIPTION
**Pull request by PaulWay**
Substantial rewrite of the parsing of the output, both to cope with the extended information from ``vgdisplay -vv`` and to reduce the number of corner cases.

This keeps the previous structure of a list of volume groups in the 'vginfo_dict' key of a 'data' dictionary.  All previous tests pass.  However, we significantly extend the keys stored, we store logical and physical volume data inside each volume group structure, and we ignore informational and verbose-status lines more easily.

Documentation for the VgDisplay parser class has also been added, with sample data and usage code.

### CODE REVIEW

File: insights/parsers/vgdisplay.py [hunk 1]
```diff
@@ -1,89 +1,184 @@
...
+        >>> vg_info.debug_info
+        ["Couldn't find device with uuid VVLmw8-e2AA-ECfW-wDPl-Vnaa-0wW1-utv7tV."]
     """
 
+    _FILTER_INFO = [
+        "Couldn't find device with uuid",
+        "physical volumes missing",
+    ]
+
+    VGDISPLAY = """
```
Comment (xiangce) [inline insights/parsers/vgdisplay.py]: Unused "VGDISPLAY"

Comment (xiangce) [inline insights/parsers/vgdisplay.py]: Others look good to me.

File: insights/parsers/vgdisplay.py [hunk 2]
```diff
@@ -1,89 +1,184 @@
...
+"""
+
     def parse_content(self, content):
         self.data = {}
         vginfo_dict_list = []
-        vginfo_dict = {}
-        debug_info = []
-        for line in get_active_lines(content):
+        self.debug_info = []
+        self.data['debug_info'] = self.debug_info
```
Comment (bfahr) [inline insights/parsers/vgdisplay.py]: Is the `data` attribute used anywhere outside of this method?  Could it be removed, or be a local var?

File: insights/parsers/vgdisplay.py (hunk 1 above)
Comment (PaulWay) [inline insights/parsers/vgdisplay.py]: Ah, of course - forgot about that.  Thanks @xiangce

File: insights/parsers/vgdisplay.py (hunk 2 above)
Comment (PaulWay) [inline insights/parsers/vgdisplay.py]: It was, but since I'm working on both of those rules I think we can remove it.  Good thinking.

File: insights/parsers/vgdisplay.py [hunk 3]
```diff
@@ -1,89 +1,154 @@
...
+
+        --- Physical volumes ---
+        PV Name               /dev/mapper/luks-96c66446-77fd-4431-9508-f6912bd84194
+        PV UUID               EfWV9V-03CX-E6zc-JkMw-yQae-wdzp-Je1KUn
+        PV Status             allocatable
+        Total PE / Free PE    118466 / 4036
+
+    Lines containing 'Couldn't find device with uuid' and 'missing physical
+    volume' are stored in a ``debug_info`` property, which is also available
+    in the ``debug_info`` key in the ``data`` dictionary.
```
Comment (bfahr) [inline insights/parsers/vgdisplay.py]: Reference to `data` attribute.

File: insights/parsers/vgdisplay.py [hunk 4]
```diff
@@ -1,89 +1,154 @@
...
     """
 
+    _FILTER_INFO = [
+        "Couldn't find device with uuid",
+        "physical volumes missing",
+    ]
+
     def parse_content(self, content):
         self.data = {}
-        vginfo_dict_list = []
```
Comment (bfahr) [inline insights/parsers/vgdisplay.py]: Reference to `self.data`.

Comment (xiangce) [inline insights/parsers/vgdisplay.py]: Removing the "self.data" breaks the running of plugin "lvm_filter".

Comment (xiangce) [inline insights/parsers/vgdisplay.py]: Oh, I see your update
